README.md

```diff
@@ -105,22 +105,24 @@ you want these variants to be:
 #### Performance Build Variants
 For accurate performance measurements, read this section!
 
-If you want to analyze performance during **local development** (note: there is a non-trivial performance impact - see caveats):
-- Recommendation: use a debuggable variant (see "local.properties helpers" below) with local Leanplum, Adjust, & Sentry API tokens: contact the front-end perf group for access to them
-- Rationale: There are numerous performance-impacting differences between debug and release variants so we need a release variant. To profile, we also need debuggable, which is disabled by default for release variants. If API tokens are not provided, the SDKs may change their behavior in non-trivial ways.
-- Caveats:
-  - debuggable has a non-trivial & variable impact on performance but is needed to take profiles.
-  - Random experiment opt-in & feature flags may impact performance (see [perf-frontend-issues#45](https://github.com/mozilla-mobile/perf-frontend-issues/issues/45) for mitigation).
-  - This is slower to build than debug builds because it does additional tasks (e.g. minification) similar to other release builds
+Before heading into performance, read this [quick guide on performance with Fenix](https://wiki.mozilla.org/Performance/How_to_get_started_on_Fenix)
```

Comment:
If we lead with this, I'm concerned they'll skip the information on build variants. I'd suggest making the build variant guidance _very short_ and then link this. e.g.

Use production variants.

Also, Read this guide for what to do with the build.

Known disabled features